op-acceptor/nat.go

```diff
@@ -69,17 +70,38 @@ func New(ctx context.Context, config *Config, version string, shutdownCallback f
 		return nil, fmt.Errorf("failed to create registry: %w", err)
 	}
 
-	envURL := os.Getenv(env.EnvURLVar)
-	if envURL == "" {
-		return nil, fmt.Errorf("devnet environment URL not provided: %s environment variable is required", env.EnvURLVar)
-	}
+	var devnetEnv *env.DevnetEnv
+	var networkName string
 
-	devnetEnv, err := env.LoadDevnetFromURL(envURL)
-	if err != nil {
-		return nil, fmt.Errorf("failed to load devnet environment from %s: %w", envURL, err)
+	// Handle different orchestrator types
+	switch config.Orchestrator {
+	case flags.OrchestratorSysext:
+		// For sysext, we need DEVNET_ENV_URL and load the devnet environment
```

Comment:
Could we incorporate this into the CLI as well?

Reply:
This has now been done; PTAL.